docs/configuration/cluster_manager/cluster_hc.rst

```diff
@@ -129,3 +132,7 @@ time, into the response.
 
 Health checks that require a more complex pattern such as send/receive/send/receive are not
 currently possible.
+
+If both "send" and "receive" are empty arrays, Envoy will perform "connect only" TCP health
```

Comment:
I would just clarify this line and remove the reference to the "send" array, since envoy enters this mode specifically when "receive" is empty.